src/passes/Print.cpp

```diff
@@ -688,12 +691,11 @@ struct PrintSExpression : public Visitor<PrintSExpression> {
     printName(curr->name);
     if (currModule && !minify) {
       // emit the function index in a comment
-      for (Index i = 0; i < currModule->functions.size(); i++) {
-        if (currModule->functions[i].get() == curr) {
-          o << " (; " << i << " ;)";
-          break;
-        }
+      if (functionIndexes.empty()) {
+        ModuleUtils::BinaryIndexes indexes(*currModule);
```

Comment:
C++11 has freed us from the classic `swap` trick; line 696 could be
`functionIndexes = std::move(indexes.functionIndexes);`
although in this case I guess it's not really any more concise.
Maybe replace both lines with
`functionIndexes = std::move(ModuleUtils::BinaryIndexes(*currModule).functionIndexes);` 
but I'm not sure you can construct an object like that.

Reply:
Looks like it is ok to construct that way.

Reply:
It causes a warning on some compilers, turns out, so I'll leave it as two lines.